### ISSUE TITLE
Outline text?

### ISSUE THREAD

**Issue by TimotheAlbouy** [shrhdk/text-to-svg#43]
Is it possible to outline the text created with this module? For example, I would like to write text in white and outlined in black. This feature is possible with the vectorize-text module.

**Comment by shrhdk** [comment]
Will the image look like this?

![image](https://user-images.githubusercontent.com/1025020/60900212-caab3700-a2a6-11e9-8c75-8b5608acd264.png)

If so, you can use attributes options like this.

```js
textToSVG.getSVG('hello', {attributes: {fill: "white", stroke: "black"}});
```

**Comment by TimotheAlbouy** [comment]
Yes that's good, thank you very much! But the strokes are outside the edges of the letters or are they a bit inside?

**Comment by shrhdk** [comment]
@TimotheAlbouy 

They are a bit inside.

If you want the stroke to be outside the edge, try the paint-order property.

https://www.oreilly.com/ideas/elegant-outlines-with-svg-paint-order

**Comment by banli17** [comment]
hi @shrhdk ，paint-order attribute does not seem to work
![image](https://user-images.githubusercontent.com/12556069/98896982-480f3900-24e5-11eb-824a-2b05902ae18d.png)
